### DISCUSSION TITLE
Weird Playlist Problem

### DISCUSSION THREAD

**Discussion by jdycus2558** [smplayer-dev/smplayer#61] [General]
I have SMPlayer installed on 2 machines (PCs): one is a desktop, the other is a laptop, both Windows 10. On the desktop, the songs in the playlist are shown in the proper order when opened a second or more times. On the laptop, the songs in the playlist appear out of order when opened the second or more times. Any clue? Thanks.

**Comment by smplayer-dev** [comment]
There's a shuffle button in the playlist. Be sure it's not enabled.
